<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BmPreferencesDialog</class>
 <widget class="QDialog" name="BmPreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brandmeister Preferences</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54); color: rgb(211, 215, 207);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>371</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabFilter">
    <attribute name="title">
     <string>Filter</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>331</height>
      </rect>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabOptions">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>210</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min. duration of transmission (sec)</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="udTransmissionDuration">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>30</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="udSilenceDuration">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>60</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>200</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min. duration of silence (sec)</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabTG">
      <attribute name="title">
       <string>TG</string>
      </attribute>
      <widget class="QListView" name="lbGroups">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>350</width>
         <height>260</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbFilterByTG">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>190</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Filter by groups</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabDxcc">
      <attribute name="title">
       <string>DXCC</string>
      </attribute>
      <widget class="QCheckBox" name="cbFilterByDxcc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>190</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Filter by DXCC</string>
       </property>
      </widget>
      <widget class="QListView" name="lbCountries">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>350</width>
         <height>260</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabCalls">
      <attribute name="title">
       <string>Calls</string>
      </attribute>
      <widget class="QCheckBox" name="cbFilterByCallsigns">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>250</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Filter by callsigns</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="tbCallsigns">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>350</width>
         <height>260</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabStorage">
    <attribute name="title">
     <string>Storage</string>
    </attribute>
    <widget class="QCheckBox" name="cbSaveToDb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Save to database</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblPathToDb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Path to database</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbPathToDb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>321</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSelectPathToDb">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>70</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbDbStorageLimit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>181</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lblDatabaseStorageLimit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Database storage limit</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabReporting">
    <attribute name="title">
     <string>Reporting</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BmPreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BmPreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
